src/Windowing/Silk.NET.Windowing.Glfw/GlfwWindow.cs

```diff
@@ -715,6 +718,16 @@ protected override void UnregisterCallbacks()
             }
         }
 
+        public new bool TopMost
```

Comment:
This will need implementing in SDL also.

Reply:
Added SDL Implementation